x-pack/legacy/plugins/infra/public/pages/metrics/layouts/aws.ts

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { InfraMetric } from '../../../graphql/types';
+import { InfraFormatterType } from '../../../lib/lib';
+import {
+  InfraMetricLayoutCreator,
+  InfraMetricLayoutSectionType,
+  InfraMetricLayoutVisualizationType,
+} from './types';
+
+export const awsLayoutCreator: InfraMetricLayoutCreator = theme => [
+  {
+    id: 'awsOverview',
+    label: 'AWS',
+    requires: ['aws'],
```

Comment:
`requires` at the top level can be removed, only the section `required` arrays are read.